### PR TITLE
Update and rename KNOWN_ISSUES.md to FAQ.md

### PR DESCRIPTION
**Pull request by Zafnok**
Renamed to FAQ since it's more conventional. Added supported GPUs section based on L1 cache size.

### CODE REVIEW

**Comment by developedby** [comment]
Thanks

File: FAQ.md [hunk 1]
```diff
@@ -43,3 +43,13 @@ Some possibilities:
 
 ### I'm getting an error of failed assertion
 - HVM currently has a bug in its conversion of f32 to f24 and it's unable to read the number 0.0. We already have a fix that we're working on.
+
+### What GPUs are supported?
+GPUs with >=128KB L1 cache per SM *should* work. This includes the following:
```
[the remark this answers is not in the document]
Comment (kings177) [reply FAQ.md:48]: it works on GPUs with L1cache of 96KB and 96KB of max shared mem per block too

[the remark this answers is not in the document]
Comment (developedby) [reply FAQ.md:48]: Wouldn't it be better to include the actual requirement here? It's like some specific capability right?

[the remark this answers is not in the document]
Comment (kings177) [reply FAQ.md:48]: yes and no, the settings are compute capability based, but for example, cc of 7.0 and 7.2 have 96KB of shared mem, but 7.5 only have 64KB, so we can't just say "any compute capability above 7.0 will work".

**Comment by developedby** [comment]
Completed in https://github.com/HigherOrderCO/Bend/pull/510

**Pull request closed without merging**